pkg/terminal/command.go

```diff
@@ -2876,6 +2876,19 @@ func exitCommand(t *Term, ctx callContext, args string) error {
 		if !t.client.IsMulticlient() {
 			return errors.New("not connected to an --accept-multiclient server")
 		}
+		bps, _ := t.client.ListBreakpoints(false)
+		numbp := 0
+		for _, bp := range bps {
+			if bp.ID >= 0 {
+				numbp++
```

Comment:
We can just short circuit here, we don't care how many breakpoints are set, just that they exist.

Reply:
Done.